docs/tests/common-setup.bash

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+_common_setup() {
+    TEST_BREW_PREFIX="$(brew --prefix)"
+    load "${TEST_BREW_PREFIX}/lib/bats-support/load.bash"
+    load "${TEST_BREW_PREFIX}/lib/bats-assert/load.bash"
+    mkdir -p ~/tmp
+    tmpdir=$(mktemp -d ~/tmp/${PROJNAME}.XXXXXX)
+    export DDEV_NO_INSTRUMENTATION=true
+    mkdir -p ${tmpdir} && cd ${tmpdir} || exit
```

Comment:
Should fail here?
```suggestion
    mkdir -p ${tmpdir} && cd ${tmpdir} || exit 1
```

Reply:
Thanks for noting that, much better with fail return.